README.md

```diff
@@ -1,37 +1,53 @@
 # h2olog
 
-[BPF](https://www.kernel.org/doc/html/latest/bpf/index.html) ([kernel doc](https://www.kernel.org/doc/Documentation/networking/filter.txt)) backed request logging client for the [H2O](https://github.com/h2o/h2o) server.
+A [varnishlog](https://varnish-cache.org/docs/trunk/reference/varnishlog.html)-like [BPF](https://www.kernel.org/doc/html/latest/bpf/index.html) ([kernel doc](https://www.kernel.org/doc/Documentation/networking/filter.txt)) backed HTTP request logging client for the [H2O](https://github.com/h2o/h2o) server.
 
-## Quickstart
+## Installing from Source
 
-Root privilege is required to interact with the BPF virtual machine.
-The log line format is: `ConnID ReqID HeaderName HeaderValue`, except the first line that represents the HTTP protocol version.
+See [requirements](#requirements) for build prerequisites.
 
 ```
-$ sudo h2olog -p $(pgrep -o h2o)
-
-888 1 RxProtocol HTTP/2.0
-888 1 RxHeader   :authority torumk.com
-888 1 RxHeader   :method GET
-888 1 RxHeader   :path /
-888 1 RxHeader   :scheme https
-888 1 TxStatus   200
-... and more ...
+$ cmake .
+$ make
+$ sudo make install
 ```
 
+For convenience, you can alternatively run the `make.sh` script.
+
 ## Requirements
 
+### For building h2olog
+
+- LLVM and clang (>= 3.7.1)
+- [BCC](https://iovisor.github.io/bcc/) (>= 0.11.0) [installed](https://github.com/iovisor/bcc/blob/master/INSTALL.md) on your system
+
```

Comment:
I suppose that `python3` and `cmake` are also required for building.

Reply:
Good catch. Python for the generator script had completely slipped off my mind. Thanks once again @syohex!